templates/borgmatic.service.j2

```diff
@@ -0,0 +1,60 @@
+# Managed by Ansible, please don't edit manually
+
+[Unit]
+Description=borgmatic backup
+Wants=backup_normal_repo.timer
+Wants=network-online.target
+After=network-online.target
+# Prevent borgmatic from running unless the machine is plugged into power. Remove this line if you
+# want to allow borgmatic to run anytime.
+ConditionACPower=true
+
+[Service]
+Type=oneshot
+User={{ borg_user }}
+ExecStart=/usr/local/bin/borgmatic -c /etc/borgmatic/{{ borgmatic_config_name }}
```

Comment:
Using an absolute path here will break things if `borg_install_method: package` is used. Same issue with the Cron job.

Reply:
`systemd[6998]: borgmatic.service: Failed to locate executable /usr/local/bin/borgmatic`